Source/Core/DolphinTool/ConvertCommand.cpp

```diff
@@ -121,35 +131,61 @@ int ConvertCommand::Main(const std::vector<std::string>& args)
       return 1;
     }
 
-    if (!CheckBlockSize(block_size_o.value(), volume->GetSize(), format))
+    if (!DiscIO::IsDiscImageBlockSizeCompatible(block_size_o.value(), format))
     {
       std::cerr << "Error: Block size is invalid" << std::endl;
       return 1;
     }
   }
 
+  if (DiscIO::IsGCZBlockSizeLegacyCompatible(block_size_o.value(), volume->GetSize()))
+  {
+    std::cerr << "Warning: For GCZs, the file size must be an integer multiple of "
+                 "the block size "
+                 "and must not be an integer multiple of the block size multiplied by 32.";
```

Comment:
This should have a mention of older versions of Dolphin. Right now it sounds like it just won't work at all.